week-1-practise/level-3/1-piping.js

```diff
@@ -11,7 +11,9 @@
 */
 
 var startingValue = 5;
-
+startingValue = multiply(startingValue, 2);
+startingValue = add(startingValue,10);
+result = format(startingValue);
```

Comment:
Can you think of a way to do all this in one line? And also not modify `startingValue`

Reply:
result = format( multiply ( startingValue , 2) + add( startingValue , 10 ) );